trace-sdk/src/main/java/io/bitrise/trace/scheduler/ScheduledJobService.java

```diff
@@ -1,49 +0,0 @@
-package io.bitrise.trace.scheduler;
```

Comment:
I removed this file as it looks like it wasn't being used by anything. Do you see any issue with this?